client/src/main/java/io/pravega/client/security/auth/JwtBody.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.security.auth;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Represents a JWT body for serialization/deserialization purposes.
+ */
+@Builder
+@Getter
+@Setter
+class JwtBody {
+
+    // See https://tools.ietf.org/html/rfc7519#page-9 for additional details about these fields.
+
+    /**
+     * The "sub" (for subject) claim of the JWT body.
+     */
+    @SerializedName("sub")
```

Comment:
What is the goal of serialization? Is it to pass data between client and server? If so, you need to use `VersionedSerializer`.

I observe this is the only place where you use `google.gson`; if true, then please remove this reference and use the `VersionedSerializer` which exists in common and which has back&forward compatibility features.

Reply:
I've modified the code now to avoid the client's dependency for Gson for JSON parsing. I now use regex to extract the expiry time from the JWT token. Gson is used for easier unit testing only. Does that work?